Gemfile

```diff
@@ -0,0 +1,18 @@
+source 'https://rubygems.org'
+
+gem 'berkshelf', '> 3.1'
```

Comment:
Minor in the scope of things, but considering the issues we have run into with the 3.x series (Faraday gzip comes to mind), I would stick to the 4.x series of berkshelf.